packages/components/select/src/useSelect.ts

```diff
@@ -296,6 +297,7 @@ export const useSelect = (props, states: States, ctx) => {
         if (props.filterable) {
           states.filteredOptionsCount = states.optionsCount
           states.query = props.remote ? '' : states.selectedLabel
+          iOSInput.value?.focus()
```

Comment:
this code is used to focus the native input when user click the suffix icon instead of directly clicking native input.

Reply:
@MrWeilian we should use 

```ts
iOSInput.value?.focus?.()
```

here to prevent incorrect invocation.

Reply:
hh, thank you so much for your suggestion, I am fixed it